pull_request_template.md

```diff
@@ -0,0 +1,9 @@
+Please ensure your pull request adheres to the following guidelines:
+
+- [ ] Use the following format: `* [owner/repo](link)`
+- [ ] Link additions should be added to the bottom of the relevant category.
+- [ ] New categories or improvements to the existing categorization are welcome.
+- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
+- [ ] Sort by alphabetical order
```

Comment:
@puneet29 Sort what?

Reply:
Please check the new pull request. Sorry for the inconvenience.